exp/production.md

```diff
@@ -0,0 +1,84 @@
+When you are planning a production deployment, you should consider the aspects that you want to customize for best results. This topic covers the most common and important requirements when you are planning to move your Anbox Cloud deployment to production. While there could be other specifically tailored needs, this topic helps you identify your basic requirements and deployment strategy.
+
+## Hardware requirements 
+
+While you can run Anbox Cloud on a single machine, you should use several machines for a production environment. Depending on your workload and the type of deployment you choose, hardware requirements can differ. See [requirements](https://anbox-cloud.io/docs/requirements) to get an idea about the kind of hardware that is required to run a full Anbox Cloud deployment with the streaming stack or a minimal core version of the deployment without the streaming stack. 
+
+You should pay special attention to the hardware requirements of applications that are used but not maintained by Anbox Cloud. Anbox Cloud Appliance has a different set of hardware requirements but we do not recommend the appliance for a production environment.
+
+## Software requirements 
+
+Anbox Cloud deployments require Ubuntu operating system. You should consider the Ubuntu OS, LXD, and Juju versions supported by Anbox Cloud for a production deployment. See [requirements](https://anbox-cloud.io/docs/requirements) to see compatible software versions.
+
+## Networking requirements 
+
+Consider the following questions to decide your networking requirements:
+
+* In your deployment, how many virtual private clouds(VPC) do you need per cluster?
+* Do you need subnets? If yes, what is the range of the subnets that you want to define?
+* Have you defined your AWS regions?
+* For each cluster, how do you want to map the subnets within the virtual private clouds into Juju spaces?
+* Where is your AMS Application Registry (AAR) deployed? How will the AMS in other clusters connect to AAR? Define your VPC peering requirements based on the number of clusters you have and where AAR is deployed.
+
+## Storage requirements
+Based on your safety protocols, think about where you would want to host your storage.
```

Comment:
We can definitely make a link to https://anbox-cloud.io/docs/requirements which links to https://etcd.io/docs/v3.5/op-guide/hardware/ and also point to https://anbox-cloud.io/docs/exp/capacity-planning to let the user calculate what kind of storage is needed